digital_asset_types/src/rpc/asset.rs

```diff
@@ -217,6 +217,10 @@ pub struct Compression {
     pub compressed: bool,
     pub data_hash: String,
     pub creator_hash: String,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub asset_data_hash: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub flags: Option<u8>,
```

Comment:
I would change the name to make it more clear what it is about and what this flag is for.